### PR TITLE
Add installation procedure for vim-plug

### PR DESCRIPTION
**Pull request by Nek-**

### CODE REVIEW

**Comment by cxw42** [comment]
Thanks for this contribution!  I think the number of options has outgrown "OR" :) .  Would you be willing to add subheaders for each item (manual, Vim8, pathogen, Vundle, vim-plug)?  E.g., 

    ...
    ### Vundle
    Use [Vundle] by ...

    ### vim-plug
    Use [vim-plug] by ...
    ...

Then I'll be happy to merge.  Much appreciated!

File: README.md [hunk 1]
```diff
@@ -41,6 +41,16 @@ OR
 
   Then remember to call `:PluginInstall`.
 
+OR
+
+- Use [vim-plug]() by adding to your `.vimrc` in your plugin section:
```
Comment (k-takata) [inline README.md:46]:
```suggestion
- Use [vim-plug][] by adding to your `.vimrc` in your plugin section:
```

Comment (Nek-) [reply README.md:46]: Can you explain that please @k-takata ?

Comment (k-takata) [reply README.md:46]: Please check the file on GitHub: https://github.com/editorconfig/editorconfig-vim/blob/add2baa345d3c5c33b0689d148df5acf4e045e6e/README.md
You may see that the link to vim-plug is broken. See also the lines 31 and 36.

**Comment by Nek-** [comment]
@cxw42 fixed.

**Comment by cxw42** [comment]
Looks good!  One comment.

File: README.md [hunk 2]
```diff
@@ -10,36 +10,48 @@ This is an [EditorConfig][] plugin for Vim. This plugin can be found on both
...
+Plugin 'editorconfig/editorconfig-vim'
+```
 
-  ```viml
-  Plugin 'editorconfig/editorconfig-vim'
-  ```
+Then remember to call `:PluginInstall`.
 
-  Then remember to call `:PluginInstall`.
+## Install with [vim-plug][]
```
Comment (cxw42) [inline README.md:46]: This should have three hashes - `###` instead of `##`.

**Comment by Nek-** [comment]
@cxw42 👍

**Comment by cxw42** [comment]
Thanks again!

**Pull request merged**